examples/interconnect_type.go

```diff
@@ -29,7 +29,7 @@ func main() {
 		fmt.Println(interconnect)
 	}
 
-	interconnect_type := utils.NewNstring("/rest/interconnect-types/a3af4d5e-7114-4e7a-a6c4-f97b707ec87c")
+	interconnect_type := utils.NewNstring("/rest/interconnects/e1242fc7-4e54-488d-90b2-2f8469a2b80a")
```

Comment:
why interconnect-types changed to interconnects..?

Reply:
i was testing something, forgot to put it back to defaults. I have fixed it, let me know